benchmarks/index.bench.ts

```diff
@@ -14,7 +15,11 @@ describe('open', () => {
   });
 
   bench('nodegit', async () => {
-    await Repository.open(gitDir);
+    await NodeGitRepository.open(gitDir);
+  });
+
+  bench('simple-git', () => {
```

Comment:
```suggestion
  bench('@napi-rs/simple-git', () => {
```
Could you please be more specific about the name, as it may overlap with other packages? (e.g. https://github.com/steveukx/git-js/tree/main/simple-git)